electron/ui/src/views/FlowsheetConfig/ConfigInput/ConfigInput.js

```diff
@@ -234,41 +234,107 @@ export default function ConfigInput(props) {
 
         }
 
-        //sorting the keys of var_sections by amount of variables into object "items"
-        let items = Object.keys(var_sections).map(function (key) {
-            return [key, var_sections[key]['num_variables']];
-        });
-        items.sort(function (first, second) {
-            return second[1] - first[1];
-        });
-        // console.log(items)
-        return var_sections
+        // sort the keys of var_sections into two groups that have as close as possible to even amount of total variables
```

Comment:
I assume this is used to calcualte spacing for the table?

Reply:
not exactly. it's to try to generate the input accordions into two columns that are close to even in height. in order to do so, we need to count how many input variables are in each category